apps/meteor/app/apps/server/bridges/listeners.ts

```
@@ -0,0 +1,366 @@
+			.executeListener(inte, data as any); // We're delegating the payload validation to the method being called
+	}
+
+	async messageEvent(inte: keyof IListenerExecutor, payload: unknown[]): Promise<boolean | IMessage | undefined> {
+		const [message, ...data] = payload as [IMessage, ...unknown[]];
+
+		const msg = await this.orch.getConverters().get('messages').convertMessage(message);
+
+		const params = ((): IAppsMessage | { message: IAppsMessage; user: IAppsUser; [key: string]: any } => {
+			switch (inte) {
+				case AppInterface.IPostMessageDeleted:
+					const [userDeleted] = data as [IUser];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)